spring-cloud-netflix-eureka-server/src/main/java/org/springframework/cloud/netflix/eureka/server/EurekaController.java

```diff
@@ -57,6 +58,9 @@
 
 	private ApplicationInfoManager applicationInfoManager;
 
+	@Autowired
+	private EurekaDeploymentProperties eurekaDeploymentProperties;
```

Comment:
The reason why constructor injection is not used is because I don’t want to break the existing test case code structure, and because I run the current test case, I found that there are some problems in the test case of this project. When these problems are solved, I very happy to use constructor injection and update relevant test code.